<ui version="4.0" >
 <class>mainwindow</class>
 <widget class="QFrame" name="mainwindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>168</height>
   </rect>
  </property>
  <property name="frameShape" >
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow" >
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridlayout" >
   <item row="0" column="0" >
    <widget class="QProgressBar" name="progressBar_upper" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>5</height>
      </size>
     </property>
     <property name="maximum" >
      <number>0</number>
     </property>
     <property name="value" >
      <number>-1</number>
     </property>
     <property name="textVisible" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" name="hboxlayout" >
     <item>
      <widget class="QLabel" name="notification_picture" >
       <property name="pixmap" >
        <pixmap>/usr/share/pnm/icons/notif.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="notification_title" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <family>UnBom</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>Notification arrived!</string>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vboxlayout" >
       <item>
        <widget class="QToolButton" name="exit_button" >
         <property name="icon" >
          <iconset>
           <normaloff>/usr/share/pnm/icons/exit.png</normaloff>/usr/share/pnm/icons/exit.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <widget class="QProgressBar" name="progressBar_lower" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>5</height>
      </size>
     </property>
     <property name="maximum" >
      <number>0</number>
     </property>
     <property name="value" >
      <number>-1</number>
     </property>
     <property name="textVisible" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QFrame" name="inner_frame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridlayout1" >
      <item row="0" column="0" >
       <widget class="QLabel" name="notification_text" >
        <property name="font" >
         <font>
          <family>UnGraphic</family>
         </font>
        </property>
        <property name="text" >
         <string>Default notification text</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <layout class="QGridLayout" name="gridlayout2" />
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
